### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by silvinux**
### What does this PR do?
Fixed whitespaces between {raw} strings. Also, I've added variable "query_controller_api_max_objects" to filetree-create role README.md. Finally, I've removed from template roles/filetree_create/templates/current_organizations.j2 the galaxy_credentials variable, which was a variable used internally and it doesnt give really value.
....
{%- raw %}
    galaxy_credentials: "{{ common_galaxy_credentials }}"

common_galaxy_credentials:
  - "Ansible Galaxy"
  - "{{ orgs }} {{ env }} Automation Hub Community Repository"
  - "{{ orgs }} {{ env }} Automation Hub Published Repository"
  - "{{ orgs }} {{ env }} Automation Hub RH Certified Repository"
{% endraw %}
....
### How should this be tested?
I've tested the  {raw} strings examples and works OK without the whitespaces.

---
controller_groups:
  - name: group1
    inventory: satlab-admin-inventory
    variables:
      some_var: some_val
      ansible_connection: local
  - name: group2
    inventory: satlab-admin-inventory
    variables:
      some_var: some_val
      ansible_connection: local
  - name: group3
    inventory: satlab-admin-inventory
    variables:
      some_var: some_val
      ansible_connection: local
      site_subnet: "{% raw %}{  % if site == 'SITE-2' %}10.200.{% else %}10.100.{% endif %}{% endraw %}"
      remote_subnet: "{% raw %}{  % if site == 'SITE-2' %}10.100.{% else %}10.200.{% endif %}{% endraw %}"
  - name: sat_lab_group
    inventory: satlab-admin-inventory
    variables:
      some_var: some_val
      ansible_connection: local
    hosts:
      - PSQL1
      - PSQL2
      - PSQL3
    children:
      - group1
      - group2
      - group3
...

---
#controller_hosts: []
#...
controller_hosts:
  - name: PSQL1
    inventory: satlab-admin-inventory
    variables:
      some_var: some_val
      ansible_connection: local
  - name: PSQL2
    inventory: satlab-admin-inventory
    variables:
      some_var: some_val
      ansible_connection: local
      site_subnet: "{% raw %}{% if site == 'SITE-2' %}10.200.{% else %}10.100.{% endif%}{% endraw %}"
      remote_subnet: "{% raw %}{% if site == 'SITE-2' %}10.100.{% else %}10.200.{% endif%}{% endraw %}"
  - name: PSQL3
    inventory: satlab-admin-inventory
    variables:
      some_var: some_val
      ansible_connection: local
...

 ansible-playbook config-controller.yml --tags inventories,hosts -e "{orgs: Global, dir_orgs_vars: orgs_vars, env: demo-dev }" -e @orgs_vars/env/demo-dev/configure_connection_controller_credentials.yml --vault-password-file ./.vault_pass.txt

Cheers

### CODE REVIEW

File: examples/configs/credential_types.yml [hunk 1]
```diff
@@ -17,9 +17,9 @@ controller_credential_types:
         - rest_password
     injectors:
       env:
-        rest_password_env: "{% raw %}{  { rest_password }}{% endraw %}"
```
Comment (sean-m-sullivan) [inline examples/configs/credential_types.yml:20]: we need to check these as they are on purpose in many places 
Without the double space, Ansible complains that rest_password is not defined, the double space is the workaround to pass the correct string to the API

[Readme part explaining this](https://github.com/redhat-cop/controller_configuration/tree/devel/roles/credential_types#formating-injectors)
[Credentials types](https://github.com/redhat-cop/controller_configuration/blob/e8576d2860e28a3f09ee2268ff131610eaf72193/roles/credential_types/tasks/main.yml#L6)
[Groups](https://github.com/redhat-cop/controller_configuration/blob/e8576d2860e28a3f09ee2268ff131610eaf72193/roles/groups/tasks/main.yml#L8)
[Hosts](https://github.com/redhat-cop/controller_configuration/blob/e8576d2860e28a3f09ee2268ff131610eaf72193/roles/hosts/tasks/main.yml#L10)
[Inventories](https://github.com/redhat-cop/controller_configuration/blob/e8576d2860e28a3f09ee2268ff131610eaf72193/roles/inventories/tasks/main.yml#L9)

File: examples/templates/vars/controller_credential_types.yml [hunk 2]
```diff
@@ -26,9 +26,9 @@ controller_credential_types:
         - ""
     injectors:
       env:
-        _HOST: "{% raw %}{  { host }}{% endraw %}"                 # DO NOT REMOVE SPACES - removing them will cause an error.
```
Comment (sean-m-sullivan) [inline examples/templates/vars/controller_credential_types.yml:29]: The comment on this was not set without reason.

File: roles/filetree_create/templates/current_organizations.j2 [hunk 3]
```diff
@@ -2,13 +2,4 @@
 controller_organizations:
   - name: "{{ current_organization.name }}"
     description: "{{ current_organization.description }}"
-{%- raw %}
```
Comment (sean-m-sullivan) [inline roles/filetree_create/templates/current_organizations.j2:5]: This looks to have already been merged can you check this?

**Comment by sean-m-sullivan** [comment]
any movement on this?

**Comment by ivarmu** [comment]
Could I help in anything with that PR?

**Comment by djdanielsson** [comment]
I think I will close this if it isn't updated by the end of the week. @silvinux

**Comment by silvinux** [comment]
> will close this if it isn't updated by t

I'm closing this PR. I will open again once I test what sean requested

**Pull request closed without merging**